bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/RawTransactionRpc.scala

```diff
@@ -88,12 +89,23 @@ trait RawTransactionRpc { self: Client =>
     bitcoindCall[Transaction]("getrawtransaction", params)
   }
 
+  /**
+    * @param maxfeerate On versions before 0.19 set to 0 if you want to enable allowhighfees
```

Comment:
Even for v0.19 `0` means enable `allowhighfees` right? I think just get rid of the first part here

Reply:
You're right, that makes more sense